.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -16,6 +13,46 @@ on:
         required: true
         default: true
         type: boolean
+      owner-subject:
+        description: 'Owner (used for creating tmp billing project)'
+        required: true
+        default: 'hermione.owner@quality.firecloud.org'
+        type: string
+      non-owner-subject:
+        description: 'Student (non-owner used for e2e testing)'
+        required: true
+        default: 'harry.potter@quality.firecloud.org'
```

Comment:
does this email have to exist? or can we try and set
non-owner-${{ github.run_attempt }}@quality.firecloud.org

Reply:
This email would need to be a registered Terra user and a valid Google user in `quality.firecloud.org` domain along with appropriate proxy groups. Otherwise, we would need to add a new user `non-owner-${{ github.run_attempt }}@quality.firecloud.org` to `quality.firecloud.org` with all groups, then register it with Sam API and accept the TOS before it can be used in testing.